xarray/core/dataset.py

```diff
@@ -124,7 +115,17 @@
     broadcast_variables,
     calculate_dimensions,
 )
-from xarray.namedarray.utils import infix_dims, is_dict_like
+from xarray.namedarray.daskmanager import DaskManager
```

Comment:
It doesn't look like we need to import this here? (Also if dask is not installed can we even import it?)